apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+| subdomain                | String  | Any subdomain part of the domain name. Ex. "sub.example"                                                                            |
+| token_name               | String  | The name of the token associated with this domain. Ex. "ExampleToken"                                                               |
+| token_standard           | String  | Aptos standard that the collection adheres to. Ex. "v1"                                                                             |
+
+### `current_ans_lookup_v2`
+
+This table maps tokens, standards, and addresses to human readable names.
+
+| Field                    | Type    | Description                                                                                                                         |
+| ------------------------ | ------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| domain                   | String  | The domain associated with this Aptos name. Ex. "example.crypto"                                                                    |
+| expiration_timestamp     | String  | Timestamp when the domain registration expires. Ex. "2024-04-17T02:14:25.68771"                                                     |
+| is_deleted               | Boolean | Indicates whether the domain registration has been deleted. Ex. False                                                               |
+| last_transaction_version | bigint  | The version number of the last transaction involving this domain. Ex. 5000000                                                       |
+| registered_address       | String  | This is an Aptos account address registered to the domain. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a" |
+| subdomain                | String  | Any subdomain part of the domain name. Ex. "sub.example"                                                                            |
+| token_name               | String  | The name of the token associated with this domain. Ex. "ExampleToken"                                                               |
+| token_standard           | String  | Aptos standard that the collection adheres to. Ex. "v1"                                                                             |
+
+# Deprecated Tables
+
+The following tables are planned for deprecation, or are already deprecated. See the notes section for any direct replacements or notes on how to migrate if you currently depend on one of these tables. Please do not use any of the below tables for production services.
+
+| Table                                                         | Deprecation Date | Notes |
+| ------------------------------------------------------------- | ---------------- | ----- |
+| address_events_summary                                        | May 15th         |       |
```

Comment:
Let's revisit these dates. I'm putting together a spreadsheet to track these. For now we can include the list of the tables but leave the dates blank.